modules/getting-started/c_sample-app.adoc

```diff
@@ -0,0 +1,11 @@
+// Metadata created by nebel
+//
+// UserStory: As an RH SSO customer, I want to complete the initial configuration of my standalone server
+
+[id="sample-app"]
+= Security for a sample application
+//If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
+
+You can use {project_name} to authenticate and authorize users of a sample application. For the sample application, you can secure a {appserver_name} servlet application. 
+
+However, you cannot use this app server as an application container. To run your Java servlet application, you must run a separate {appserver_name} instance.
```

Comment:
I would make this part of the previous sentence, rather than "However, you cannot use this app server as an application container." starting the next paragraph on its own.

e.g. "For the sample application, you can secure a {appserver_name} servlet application, however you cannot use this app server as an application container."